JavaFXUserInterface/src/main/java/Controllers/SystemInfoController.java

```diff
@@ -188,15 +201,19 @@ else if (series.getData().size() != measurements.size()) {
      * @author Jeff
      */
     public void attachEvents() {
-        systemMenuBtn.setOnMouseClicked(mouseEvent -> uiUtilities.changeScene(mouseEvent, "/Systems"));
-        //Attach link to systemTypeMenuBtn to go to Utilities.SystemTypeList.fxml
+        systemMenuBtn.setOnMouseClicked(mouseEvent -> {
+            timelines.forEach(Timeline::stop);
+            uiUtilities.changeScene(mouseEvent, "/Systems");
+        });
+        //Attach link to systemTypeMenuBtn to go to SystemTypeList.fxml
         systemTypeMenuBtn.setOnMouseClicked(mouseEvent -> uiUtilities.changeScene(mouseEvent, "/SystemTypeList"));
```

Comment:
Timelines need to be stopped here.

Reply:
this is what  timelines.forEach(Timeline::stop); does